scripts/bikes.js

```diff
@@ -444,10 +451,19 @@ function openRateTripMenu(tripObj) {
 
 async function rateTrip(tripCode, tripCost) {
 	// Get the selected input for the stars
-	const tripRating = Number(document.querySelector(`input[type="radio"]:checked`).value);
+	let starsInput = document.querySelector(`input[type="radio"]:checked`);
+	let tripRating;
+	if (starsInput) {
+		tripRating = Number(starsInput.value);
 
```

Comment:
neste caso podes fazer isso porque no caso de startsInput ser falsy dás return então isto vai sempre ter o valor real
```suggestion
	const tripRating = Number(starsInput?.value);
	if (starsInput) {
```